arches/app/media/js/views/resource/editor.js

```diff
@@ -296,10 +296,24 @@ define([
                     vm.alert(new JsonErrorAlertViewModel('ep-alert-red', err.responseJSON));
                 },
                 success: function() {
-                    window.location.reload();
+                    window.location.reload();  // reload is important here, for enforcing a report redirect on an unpermissioned user 
                 }
             });
         },
+        onSaveSuccess: function() {
```

Comment:
Nice work improving the save interaction to let the user stay on the card without a redirect in most cases.

I'll open a follow-up ticket for whether to disable the lifecycle change buttons when the card is dirty.